### ISSUE TITLE
Update Stripe prices

### ISSUE THREAD

**Issue by UndercoverNL** [stripe/stripe-php#1122]
So I am making my first production payment system using stripe in Laravel. I am NOT using the Laravel integrated cashier Stripe functions. Since the Stripe SDK API ones have way more possibilities.

First I want to make everything to be possible through the website / API to make it possible not using the Stripe dashboard for any non-financial edits. (Which is the idea of using an API to me.)

Everything worked out fine until now. Accounts are getting saved as a Stripe customer, payment methods can be saved, and when a product is made on the site it gets created on Stripe to.

Now I run intro a problem. When an admin creates a product on the site, it creates on Stripe to. And the Price provided by the admin will get added to the created Stripe product.

But of course there is a function in the site to make it possible to edit products. When a product gets edited on the site, the product on Stripe will get edited to. But it seems there is no possibility to edit the Stripe Price. There is a function `$stripe->prices->update(` which can be found here https://stripe.com/docs/api/prices/update, but this function seems to be completely useless to me since there is no parameter for updating the unit_amount.

So to summarize my question: How can you update the unit_amount of a Stripe price.

Thanks in advance!

**Comment by remi-stripe** [comment]
@UndercoverNL Github issues should be limited to bugs with the library itself. Your question is more an integration question and you should instead talk to our support team: https://support.stripe.com/contact

Just to clarify though, it is impossible to edit a Price's amount, you have to simply create a new one with the new amount instead
